cmd/registry/registryList.go

```diff
@@ -87,10 +87,14 @@ func NewCmdRegistryList() *cobra.Command {
 
 			util.PrintNodes(existingNodes, registryListFlags.output,
 				headers, util.NodePrinterFunc(func(tabwriter *tabwriter.Writer, node *k3d.Node) {
+					cluster := "*"
+					if _, ok := node.Labels[k3d.LabelClusterName]); ok {
+						cluster = node.Labels[k3d.LabelClusterName])
+					}
 					fmt.Fprintf(tabwriter, "%s\t%s\t%s\n",
 						strings.TrimPrefix(node.Name, "/"),
 						string(node.Role),
-						node.Labels[k3d.LabelClusterName])
+						cluster
 				}))
 		},
```

Comment:
Just two more things: As I wrote my suggestions in GitHub, I didn't have linting. But to make this work, we'll have to change this to:
```Go
util.PrintNodes(existingNodes, registryListFlags.output,
				headers, util.NodePrinterFunc(func(tabwriter *tabwriter.Writer, node *k3d.Node) {
					cluster := "*"
					if _, ok := node.Labels[k3d.LabelClusterName]; ok {
						cluster = node.Labels[k3d.LabelClusterName]
					}
					fmt.Fprintf(tabwriter, "%s\t%s\t%s\n",
						strings.TrimPrefix(node.Name, "/"),
						string(node.Role),
						cluster,
					)
				}),
			)
```
This removes a bracket and adds a comma and does some structuring.

Also, there's still the `PORT` in the headers, which is not in the output itself, making it look like this:
```shell
$ k3d registry list

NAME                       PORT       ROLE   CLUSTER
k3d-k3s-default-registry   registry   *
k3d-myreg                  registry   *
```